CFD-366 groups global overview

MPParsley opened the pull request:
Groups overview (including search & filter by interests & filter by my groups).

![groups overview](https://cloud.githubusercontent.com/assets/1823998/6783351/906acd42-d179-11e4-8a04-968d2f51f71a.png)

zero2one commented:
Can you put the group title on top of the image and details column ([see wireframes](https://projects.invisionapp.com/d/main#/console/1420103/65349064/preview)).
Put the group access type icon in front of the group name.

![group-teaser](https://cloud.githubusercontent.com/assets/133124/6790220/6f94d860-d1a6-11e4-9a13-1950ffeff96b.png)

MPParsley commented:
added ds layout, group type icon field, moved title to layout header, added group access type icon in front of title
![groups overview](https://cloud.githubusercontent.com/assets/1823998/6798153/7f53419c-d20a-11e4-89ac-5fcc1c73c633.png)
